_directory/data/codedivision/data.yml

```diff
@@ -0,0 +1,31 @@
+---
+# name (required)
+# may be your real name or your nick
+name: Code Division GmbH
```

Comment:
Please use your real or nickname, nomcompany.
Feel free to add several files, one for each member of your team.